README.md

```diff
@@ -1,13 +1,45 @@
-# Shopify App CLI 
+# [Shopify App CLI](https://shopify.dev/tools/cli)
 
 [![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md)[![Build Status](https://travis-ci.com/Shopify/shopify-app-cli.svg?token=qtPazgjyosjEEgxgq7VZ&branch=master)](https://travis-ci.com/Shopify/shopify-app-cli)
 
-Shopify App CLI helps you build Shopify apps faster. It automates many common tasks in the development process and lets you quickly add popular features, such as billing and webhooks.
+Shopify App CLI helps you build Shopify apps faster. It quickly scaffolds Node.js and Ruby on Rails apps and automates many common development tasks.
```

Comment:
Scaffolding is a named feature in Rails, but I'm not sure if that term is known elsewhere – someone with Node experience may be able to confirm.

Reply:
('generates' may be more neutral)